KarhooUISDKTestUtils/Mocks/Common/MockDataFormatterType.swift

```diff
@@ -63,10 +63,12 @@ public class MockDateFormatterType: DateFormatterType {
         clockTimeSet = date
         return clockTimeReturnString
     }
-
-    public var customDateFormatToReturn = ""
-    public func display(_ date: Date?, dateStyle: DateFormatter.Style, timeStyle: DateFormatter.Style) -> String {
-        fullTimeSet = date
-        return customDateFormatToReturn
+    
+    public func display(
+        _ date: Date?,
+        dateStyle: DateFormatter.Style,
+        timeStyle: DateFormatter.Style
+    ) -> String {
+        TestUtil.getRandomString()
```

Comment:
You can not return random string, cuz it won't be working with snapshot tests

Reply:
If you take a look at this class every "display" method return random string. I can hardcoded some value, but rather we should create ticket and check why/how it worked for now.